test/vector2_test.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:flame/extensions/vector2.dart';
+import 'package:flame/components.dart';
```

Comment:
Not that it matters, but maybe this should import the `extensions.dart` instead?